### PR TITLE
[353575] Renewal note field is prefilled from order template (thunderjet) (TaaS)

### PR DESCRIPTION
**Pull request by dnovikau**
[C353575](https://foliotest.testrail.io/index.php?/cases/view/353575)
***
[Allure report](https://jenkins-aws.indexdata.com/job/folioRancher/job/folioTestingTools/job/runCypressTests/5490/allure)
***

Screenshot 2023-11-16 at 12 46 48

### CODE REVIEW

File: cypress/e2e/orders/renewal-field-prefilled-from-template.cy.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+      // Select template name from Precondition in "Template name" dropdown
+      OrderEditForm.selectOrderTemplate(testData.orderTemplate.templateName);
+
+      // Fill all required fields with valid values and click "Save & close" button
+      OrderEditForm.selectVendorByName(testData.organization.name);
+      OrderEditForm.clickSaveButton();
+
+      // Click "Actions" button, Select "Add PO line" option
+      const OrderLineEditForm = OrderDetails.selectAddPOLine();
```
Comment (manowarqqq) [inline cypress/e2e/orders/renewal-field-prefilled-from-template.cy.js:62]: No assertion from step 4
"Renewal note" field in "Ongoing order information" accordion contains prefilled text from template

File: cypress/support/fragments/orders/orderEditForm.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  including,
+  matching,
+} from '../../../../interactors';
+import OrderStates from './orderStates';
+import SearchHelper from '../finance/financeHelper';
+import InteractorsTools from '../../utils/interactorsTools';
+
+const orderEditFormRoot = Section({ id: 'pane-poForm' });
+
+const cancelButtom = Button('Cancel');
```
Comment (manowarqqq) [inline cypress/support/fragments/orders/orderEditForm.js:15]: typo Button

File: cypress/support/fragments/orders/orderEditForm.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+const saveButtom = Button('Save & close');
```
Comment (manowarqqq) [inline cypress/support/fragments/orders/orderEditForm.js:16]: typo Button

File: cypress/support/fragments/orders/orders.js [hunk 4]
```diff
@@ -237,6 +238,14 @@ export default {
     cy.do([Button('Receive').click(), PaneHeader('Receiving').is({ visible: true })]);
   },
 
+  clickCreateNewOrder() {
+    cy.do([actionsButton.click(), newButton.click()]);
+
```
Comment (manowarqqq) [inline cypress/support/fragments/orders/orders.js:243]: Please, remove empty line

File: cypress/e2e/orders/renewal-field-prefilled-from-template.cy.js (hunk 1 above)
Comment (dnovikau) [reply cypress/e2e/orders/renewal-field-prefilled-from-template.cy.js:62]: Fixed

File: cypress/support/fragments/orders/orderEditForm.js (hunk 2 above)
Comment (dnovikau) [reply cypress/support/fragments/orders/orderEditForm.js]: Fixed

File: cypress/support/fragments/orders/orderEditForm.js (hunk 3 above)
Comment (dnovikau) [reply cypress/support/fragments/orders/orderEditForm.js]: Fixed

File: cypress/support/fragments/orders/orders.js (hunk 4 above)
Comment (dnovikau) [reply cypress/support/fragments/orders/orders.js]: Fixed

**Review by manowarqqq — APPROVED** [review]

**Review by IhorBohdan — APPROVED** [review]

**Pull request merged**